src/js/app.js

```diff
@@ -0,0 +1,67 @@
+$(document).ready(function() {
+
+
+
+
+$('#searchBtn').on('click', function(event) {
```

Comment:
Note that for clicking on buttons, there's no default action. There is for forms though!

You may want to put an id on the form, then use a `'submit'` event instead. That way, when you press enter in the input field, it'll start the search.